kong/plugins/influxdb-log/handler.lua

```diff
@@ -0,0 +1,105 @@
+local HttpLogHandler = require "kong.plugins.http-log.handler"
+
+local InfluxdbLogHandler = HttpLogHandler:extend()
```

Comment:
`serialize ` will be invoke in the log-function

```
function HttpLogHandler:log(conf)
  HttpLogHandler.super.log(self)

  local ok, err = ngx.timer.at(0, log, conf, self:serialize(ngx, conf), self._name)
  ...
```

in addition, there are two reasons, about not use basic_serializer

* context data need covert to an influxdb-line-protocol body
* some of the data is not suitable for storage in the influxdb